### DISCUSSION TITLE
Need to update phone and email of user

### DISCUSSION THREAD

**Discussion by Xitija** [Sunbird-Lern/Community#66] [Query] [answered]
Hello All,

I am trying to update the user mail and phone by using the 
API - /api/user/v3/update 
I am getting unauthorized when I am trying to update the user

I want to know whether I can call this API with admin access.

cc : @komalm

**Comment by rhwarrier** [comment, marked as answer]
That should be possible if you have admin rights for the same tenant. 
@reshmi-nair  can you pls confirm ?

**Reply by reshmi-nair** [reply]
@Xitija Currently admin cannot update another user's email and phone using user update API. /v1/user/upload API can be used to update the email and phone of an existing user by giving userid.
